pkg/3scale/amp/auto-generated-templates/amp/amp-eval-s3.yml

```diff
@@ -1623,6 +1623,8 @@ objects:
             - bash
             - -c
             - bundle exec rake boot openshift:deploy MASTER_ACCESS_TOKEN="${MASTER_ACCESS_TOKEN}"
+              && bundle exec rake services:create_backend_apis services:update_metric_owners
```

Comment:
The new tasks have been added in a new bundle exec rake command due to it seems when used in the same bundle exec rake it failed. After commenting this with @guicassolato it seems that bundle exec rake loads everything in the same scope and the information that first tasks update is not visible to the others. That made the tasks fails. Changing it to separate tasks seems to have fixed the problem